### DISCUSSION TITLE
Issues with full screen when trying to present Quarto revealjs via Zoom

### DISCUSSION THREAD

**Discussion by pgseye** [quarto-dev/quarto-cli#7839] [Q&A]
### Description

I'm just discovering how nice a Quarto revealjs presentation can be and am trying to move away from manually making slides in Powerpoint to using this new format for talks. 

I'm wondering what are good practices for presenting via Zoom if one wants to use Speaker mode? It seems the default behaviour in Quarto is that you can enter Full screen (pressing f) but then if you try and enter Speaker view (by pressing s), full screen turns itself off. 

I can't figure out a way to present in full screen and also retain the speaker window - the closest I have come to a solution is to select the Safari window containing the rendered presentation in Zoom and then have the Speaker window over the top, but the Safari menu still shows. Also attempting this from a laptop, so don't have multiple screens to play with. 

Is there any way to do this? Otherwise how do people present using Quarto via Zoom?

**Comment by pgseye** [comment]
I just discovered multiplex - that is pretty cool, but I'm not sure how much of a substitute it can be for a direct share. Slide changes appear to stay in sync but I didn't have any further interactive control - e.g.  changing through tabsets, moving through rows of a dataframe etc.

I am keen to hear how other people present using Quarto revealjs.

**Comment by cderv** [comment]
How are you sharing your slide over zoom ? do you share Screen or do you share browser application only ? 

Speaker View works best if you have two screen 
- One to present from that is shared to viewer
- Another one with your speaker view on the side

If you share over zoom your one and only screen, you won't be able to have speaker view and full screen, because zoom is sharing exactly what your are seeing yourself to the audience. 
If you share via Zoom only one application on your computer, it may work, but not full screen mode from revealjs, probably only full screen from browser (example for [Chrome](https://www.lifewire.com/activate-full-screen-mode-google-chrome-4103634)) - this may work as this is removing nav bar and all. 

Overall, this is not about Quarto. You'll have the issue with anything you want to present full screen. You need to find what works for you, or tools that helps you share part of your screen only - [Zoom should allow that too](https://support.zoom.com/hc/en/article)

**Comment by pgseye** [comment]
Thanks. Ok that makes sense. I normally share the browser application. I thought I was missing something but clearly there are limitations to what can be done, especially when just using a laptop.
